### PR TITLE
add contrib exsrvmonit.py script to monitor active electrumx servers

### PR DESCRIPTION
**Pull request by zebra-lucky**
- exsrvmonit.py to run from cron
- spec and Makefile to build binary to run without requirements (to simplify cron run)

### CODE REVIEW

File: contrib/dash/exsrvmonit/exsrvmonit.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    return res
+
+
+def read_recent_file():
+    try:
+        with open(RECENT_FNAME, 'r', encoding='utf-8') as f:
+            data = f.read()
+            return json.loads(data)
+    except Exception:
+        return []
```
Comment (jardev) [inline contrib/dash/exsrvmonit/exsrvmonit.py]: Maybe add some logging there? If it fails silently it's gonna be very hard to debug

File: contrib/dash/exsrvmonit/exsrvmonit.py [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+
+def save_recent_file(recent):
+    s = json.dumps(recent, indent=4, sort_keys=True)
+    try:
+        with open(RECENT_FNAME, 'w', encoding='utf-8') as f:
+            f.write(s)
+    except Exception:
+        pass
```
Comment (jardev) [inline contrib/dash/exsrvmonit/exsrvmonit.py]: Same here. Logging? Somehow the situation must be processed and not just silently passed.

File: contrib/dash/exsrvmonit/exsrvmonit.py (hunk 1 above)
Comment (zebra-lucky) [reply contrib/dash/exsrvmonit/exsrvmonit.py]: fixed in 44daed8b

File: contrib/dash/exsrvmonit/exsrvmonit.py (hunk 2 above)
Comment (zebra-lucky) [reply contrib/dash/exsrvmonit/exsrvmonit.py]: fixed in 44daed8b

**Pull request merged**